### ISSUE TITLE
Redmine 3.4.0 - wrong number of arguments

### ISSUE THREAD

**Issue by peterbeck** [noshutdown-ru/vault#11]
Hi there, 

should this plugin work with RM 3.4.0 ? I have recently upgraded to 3.4.0, but somehow it throws this error now:

```
ActionView::Template::Error (wrong number of arguments (1 for 0)):
     97:   });
     98: </script>
     99: 
    100: <%= context_menu project_keys_context_menus_path(@project) %>
    101: 
    102: <script type="text/javascript">
    103:   $('a[href^="http://"]').attr('target','_blank');
  app/helpers/application_helper.rb:1299:in `context_menu'
  plugins/vault/app/views/keys/index.html.erb:100:in `_plugins_vault_app_views_keys_index_html_erb__2715060936850346617_13658000'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```
I did  a fresh clone from github and followed the instructions, but maybe I have missed something ?

Regards
Peter

**Comment by noshutdown-ru-user** [comment]
we will look on it, thx for report

**Comment by noshutdown-ru-user** [comment]
you can faster replace string from `<%= context_menu project_keys_context_menus_path(@project) %>` to   `<%= context_menu %>`

**Comment by peterbeck** [comment]
thanks for the fast reply - changing to context_menu really made it working on 3.4 as far as I can see.

**Comment by pdostal** [comment]
As I run the application in production mode I had to also restart it, but thanks @noshutdown-ru-user for the hint. It's a bit pitty this plugin's not very active last half a year.

**Comment by noshutdown-ru-user** [comment]
yes we know will return work on this plugin next 3 month

**Comment by noshutdown-ru-user** [comment]
Fixed in version 0.3.2 https://noshutdown.ru/redmine-plugins-vault/
